bftengine/src/bftengine/messages/PrePrepareMsg.cpp

```diff
@@ -330,5 +331,14 @@ std::string PrePrepareMsg::getCid() const {
   return std::string(this->body() + this->payloadShift() - b()->batchCidLength, b()->batchCidLength);
 }
 
+void PrePrepareMsg::updateCid(SeqNum s) {
```

Comment:
Having two different functions for CID changes - 'set' and 'update' confuses. I propose to replace 'set' functionality with 'update' and this way to keep only one function.